pkg/ipamd/ipamd.go

```diff
@@ -1148,6 +1167,21 @@ func getMinimumIPTarget() int {
 	return noMinimumIPTarget
 }
 
+func disablingENIProvisioning() bool {
```

Comment:
This function could be simplified using :

https://github.com/aws/amazon-vpc-cni-k8s/blob/8ffcd8a6168295df4ab2fd62d0cc3e3b5e4bae3d/pkg/ipamd/introspect.go#L203